src/tools/rest-frontend/resources/structure.json

```diff
@@ -0,0 +1,1896 @@
+                        "type": "section"
+                    },
+                    {
+                        "name": "Import Configuration",
+                        "type": "file",
+                        "options": {
+                            "path": "doc/tutorials/import.md"
+                        },
+                        "slug": "import-configuration"
+                    },
+                    {
+                        "name": "Export Configuration",
+                        "type": "file",
+                        "options": {
+                            "path": "doc/tutorials/export.md"
+                        },
+                        "slug": "export-configuration"
+                    },
+                    {
+                        "name": "Intercept Environment",
+                        "type": "file",
+                        "options": {
+                            "path": "src/bindings/intercept/env/README.md"
+                        },
+                        "slug": "intercept-environment"
+                    },
+                    {
+                        "name": "Intercept File System",
+                        "type": "file",
+                        "options": {
+                            "path": "src/bindings/intercept/fs/README.md"
+                        },
+                        "slug": "intercept-file-system"
+                    },
+                    {
+                        "name": "Merge Configuration",
+                        "type": "file",
+                        "options": {
+                            "path": "doc/tutorials/merge.md"
+                        },
+                        "slug": "merge-configuration"
+                    },
+                    {
+                        "name": "Validate Configuration",
+                        "type": "file",
+                        "options": {
+                            "path": "doc/tutorials/validation.md"
+                        },
+                        "slug": "validate-configuration"
+                    },
+                    {
+                        "name": "Package Maintainers",
+                        "type": "section"
+                    },
+                    {
+                        "name": "Merge Configuration Files",
+                        "type": "file",
+                        "options": {
+                            "path": "doc/tutorials/elektra-merge-integration.md"
+                        },
+                        "slug": "merge-configuration-files"
+                    }
+                ]
+            },
+            {
+                "name": "Plugins",
+                "type": "listfiles",
+                "ref": "plugins",
+                "children": [
+                    {
+                        "name": "README",
+                        "type": "file",
+                        "options": {
+                            "path": "src/plugins/README.md"
```

Comment:
You can delete the comments to this file when you read it. I'll remove the file from the repo.